src/network.ts

```diff
@@ -106,8 +106,7 @@ export const startConnectivityCheck = _.once(
 );
 
 export function enableConnectivityCheck(enable: boolean) {
-	const boolEnable = checkTruthy(enable);
-	enable = boolEnable != null ? boolEnable : true;
+	enable = checkTruthy(enable);
```

Comment:
Should this be `!checkFalsey`? I'm wondering because of the previous 

```
enable = boolEnable != null ? boolEnable : true; 
```

Reply:
Considering that the previous code defaulted to true then using `!checkFalsey` would follow that philosophy because it will always be true unless the value explicitly matches a false value. I can make this change.

Reply:
@pipex added change.